src/supertux/menu/editor_levelset_select_menu.hpp

```diff
@@ -21,6 +21,7 @@
 
 class EditorLevelsetSelectMenu final : public Menu
 {
+  friend class EditorDeleteLevelsetMenu;
```

Comment:
Please don't use `friend`; add the function needed as appropriate.

`EditorLevelsetSelectMenu` is a menu class; it would be preferable to find an appropriate utility class to handle deleting a world.